Add the rrc_msg_type to the description of the procedures. (#49)
tlswg/dtls-rrc: draft-ietf-tls-dtls-rrc.md

   type `path_challenge` and places the unpredictable cookie into the message.
1. The message is sent to the observed new address and a timer T (see
   {{timer-choice}}) is started.

> 1. The peer endpoint, after successfully cryptographically verifying the received
>    `return_routability_check` message of
>    type `path_challenge` responds by echoing the cookie value in a

Reviewer: ```suggestion
1. The peer endpoint cryptographically verifies the received
   `return_routability_check` message of
   type `path_challenge` and responds by echoing the cookie value in a
```
